docs/document/content/user-manual/shardingsphere-jdbc/spring-boot-starter/rules/mix.en.md

```diff
@@ -8,7 +8,7 @@ weight = 6
 # data source configuration
 spring.shardingsphere.datasource.names= write-ds0,write-ds1,write-ds0-read0,write-ds1-read0
 
-spring.shardingsphere.datasource.write-ds0.url= # Database URL connection
+spring.shardingsphere.datasource.write-ds0.jdbc-url= # Database URL connection
```

Comment:
Thank you for your excellent work, there may be more places than just this file, I hope you can search for a match and modify it as much as possible.

Reply:
Sure I will modify it, thank you for your consideration

Reply:
Hi @totalo I have find some more places here in file [mix.cn.md](https://github.com/apache/shardingsphere/blob/master/docs/document/content/user-manual/shardingsphere-jdbc/spring-boot-starter/rules/mix.cn.md) could you open a new issue so that I can work on it

Reply:
> Hi @totalo I have find some more places here in file [mix.cn.md](https://github.com/apache/shardingsphere/blob/master/docs/document/content/user-manual/shardingsphere-jdbc/spring-boot-starter/rules/mix.cn.md) could you open a new issue so that I can work on it

hi @Amulya-coder, nice work, You can follow https://github.com/apache/shardingsphere/issues/16203.